app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -57,6 +72,10 @@
 public class SubscriptionFragment extends Fragment {
 
     public static final String TAG = "SubscriptionFragment";
+    public static final String PREF_AUTO_DOWNLOAD = "auto_download_pref";
+    public static final String PREF_KEEP_UPDATED = "keep_updated_pref";
+    public static final String PREF_AUTO_DELETE = "auto_delete_pref";
```

Comment:
I think those constants should mention that they are related to filtering. `FILTER_AUTO_DELETE`? Do they need to be public?